.golangci.yml

```diff
@@ -13,16 +13,13 @@ linters-settings:
 linters:
   disable-all: true
   enable:
-    - deadcode    #Default linter
```

Comment:
why remove these 3 checks ? are they activated by default now or is there another reason ?

Reply:
Because this is now deleted in golangci, because they are no longer maintained